geopandas/plotting.py

```diff
@@ -86,6 +86,43 @@ def _expand_kwargs(kwargs, multiindex):
             kwargs[att] = np.take(value, multiindex, axis=0)
 
 
+def PolygonPatch(polygon, **kwargs):
+    """Constructs a matplotlib patch from a Polygon geometry
+
+    The `kwargs` are those supported by the matplotlib.patches.Polygon class
+    constructor. Returns an instance of matplotlib.patches.PathPatch.
+
+    Example (using Shapely Point and a matplotlib axes):
+
+    >>> b = shapely.geometry.Point(0, 0).buffer(1.0)
+    >>> patch = PolygonPatch(b, fc='blue', ec='blue', alpha=0.5)
+    >>> ax.add_patch(patch)
+
+    This code of this function is based on the descartes package by
+    Sean Gillies (BSD license, https://pypi.org/project/descartes).
+    """
+    from matplotlib.patches import PathPatch
+    from matplotlib.path import Path
+
+    def coding(ob):
+        # The codes will be all "LINETO" commands, except for "MOVETO"s at the
+        # beginning of each subpath
+        n = len(getattr(ob, "coords", None) or ob)
+        vals = np.ones(n, dtype=Path.code_type) * Path.LINETO
```

Comment:
Is there a reason to use `np.ones(shape, dtype)*value` and not `np.full(shape, value, dtype)`?

Reply:
No reason than that it was copied from descartes, which might have been written before np.full existed ;) 
But good catch, thanks, will update!